src/main/pg/gyrodev.c

```diff
@@ -446,6 +446,56 @@ void pgResetFn_gyroDeviceConfig(gyroDeviceConfig_t *devconf)
 #endif
 #endif // GYRO_COUNT 3
 
+#if GYRO_COUNT > 3
+    devconf[3].index = 3;
+#ifdef GYRO_4_SPI_INSTANCE
+    // TODO: CLKIN gyro 4 on separate pin is not supported yet. need to implement it
+    gyroResetSpiDeviceConfig(&devconf[3], GYRO_4_SPI_INSTANCE, IO_TAG(GYRO_4_CS_PIN), IO_TAG(GYRO_4_EXTI_PIN), IO_TAG(GYRO_4_CLKIN_PIN), GYRO_4_ALIGN, GYRO_4_CUSTOM_ALIGN);
+#else
+    devconf[3].busType = BUS_TYPE_NONE;
+#endif
+#endif // GYRO_COUNT 4
+
+#if GYRO_COUNT > 4
```

Comment:
Four initialized gyros may be enough - user can configure more using CLI

Reply:
I'll wait and see what others say before I remove it, but thanks for looking over this code again. I'm hoping that this can essentially be the final version (still need to sort out MSP a bit).

Reply:
@nerdCopter yes I've seen that FC. It would be able to fuse 2 of those gyros together at a time with this code (2 have the same scale and update rate).
How this is currently coded it is currently a custom define that would exist in each boards conf.h file. All that this code going up to 8 IMU does is make it so that if someone made an FC with 4+ IMU that they could initialize then in their conf.h file rather than making the users configure them themselves in the cli.